hermes-console/static/partials/modal/editTopic.html

```diff
@@ -57,6 +57,15 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit topic:</small
                     </select>
                 </div>
             </div>
+            <div class="form-group {{topicForm.topicMaxMessageSize.$valid ? '' : 'has-error'}}">
+                <label for="topicMaxMessageSize" class="col-md-3 control-label">Max message size</label>
+                <div class="col-md-9">
+                    <div class="input-group">
+                        <input type="number" required class="form-control" id="topicMaxMessageSize" name="topicMaxMessageSize" ng-model="topic.maxMessageSize" placeholder="how large messages are for this topic?"/>
```

Comment:
I was trying not to use words like "maximum message size" to describe field called "maximum message size"